File: FrameworkView_Controller_ModelCSV_Mod/src/main/webapp/assets/js/Validator.js
/**
 * Aplica validaciones de expresiones regulares sobre contenido.
 * @author jose.inestroza@unah.edu.hn
 * @version 0.1.0
 */
 
class Validator{
	constructor(){}
	
	isEmail(text){
		if(`${text}`.trim().match(/^[\w.]+@[\w]+(\.[a-zA-Z]{2,})+$/)){
			return true;
		}
		return false;
	}
	
	removeHTML(text){
		return `${text}`.trim().replace(/<[^>]+>/g,"");
	}
	
	removeSpecialCharacters(text){
		return `${text}`.trim().replace(/[^\w \t.,-:áéíóúüñÁÉÍÓÚÜñÑ]+/g,"");
	}
}

File: Proyecto Avance 3 Kenneth Pineda 20201002741/Código/DocumentManager_Avance3_KennethPineda/src/main/java/programLibraries/Validator.java
package programLibraries;

public class Validator {
	public boolean isEmail(String text) {
		if(text.trim().matches("[\\w.]+@[\\w]+(\\.[a-zA-Z]{2,})+")) {
			return true;
		}
		return false;
	}
	
	public String removeHTML(String text) {
		return text.trim().replaceAll("<[^>]+>", "");
	}
	
	public String removeSpecialCharacters(String text) {
		return text.trim().replaceAll("[^\\w \\t.,-:áéíóúüñÁÉÍÓÚÜñÑ]+", "");
	}
}
